src/hotspot/share/runtime/thread.cpp

```diff
@@ -1613,6 +1614,9 @@ void JavaThread::check_and_handle_async_exceptions() {
   }
 
   if (!clear_async_exception_condition()) {
+    if ((_suspend_flags & _async_delivery_disabled) != 0) {
```

Comment:
Hmmm... @robehn is going to want to know about a new use of `_suspend_flags`.

Reply:
Yes. If we stop using them for async exceptions then the delivery_disabled bit will also have to be relocated. For now at least it is free to use.